changelog/std-random-MersenneTwisterEngine.dd

```diff
@@ -2,8 +2,13 @@
 matches the C++11 standard (adding two new template parameters, an extra
 tempering parameter `d` and the initialization multiplier `f`).
 
+Handling of the word size `w` has been fixed so that the generator will
+now properly handle cases where it is less than the number of bits in
+the chosen `UIntType`.  This has been validated against the behaviour
+of the C++11 standard implementation.
```

Comment:
Ah, good point.  Obviously I compared against an implementation of the C++11 standard.  However the standard itself does mandate the expected behaviour (which is really only re-stating the expected behaviour of the algorithm itself).  See p. 899 (numbers; p. 913 in your PDF browser) of http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2012/n3337.pdf.